### PR TITLE
ENG-2371: fixing form widget clone page (also fixes ENG-2373)

### PR DESCRIPTION
**Pull request by gReis89**
this one address ENG-2371 and ENG-2373 bugs.

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widgets/clone/CloneWidgetPage.js
Comment (gidesan) [inline src/ui/widgets/clone/CloneWidgetPage.js:20]: I'd use "Clone" instead of "Add" in the `className`

File: src/ui/widgets/config/forms/NavigationBarConfigForm.js
Comment (gidesan) [inline src/ui/widgets/config/forms/NavigationBarConfigForm.js:20]: I'm not a fan of those config flags. 
On top of that, I `WidgetForm` already has info about "mode" which could be new | edit | clone.
You could stick to that solution. Not the best possible one, but cleaner than having scattered flags from different sources of truth.

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/ui/widgets/config/forms/NavigationBarConfigForm.js
Comment (gReis89) [reply src/ui/widgets/config/forms/NavigationBarConfigForm.js]: this page is only loaded for clone mode, there is a handler in WidgetForm that ensure it will be only loaded for this mode:
```
    const NativeWidgetConfigForm = parentWidget
      && mode === MODE_CLONE
      && getAppBuilderWidgetForm(parentWidget, true);
```

also, this page is loaded dynamically by `getAppBuilderWidgetForm` so I couldn't pass down the `mode` prop to this component, that's why I'm getting it from the URL

File: src/ui/widgets/config/forms/NavigationBarConfigForm.js
Comment (gReis89) [inline src/ui/widgets/config/forms/NavigationBarConfigForm.js:53]: have u tested this changes @chalagashvili ?

As far as I know, `mode` is not injected on NavigationBarConfigForm by default(I tried to inject it via props, but it is loaded dynamically by a method and it didn't work).

File: src/ui/widgets/config/forms/NavigationBarConfigForm.js
Comment (gReis89) [reply src/ui/widgets/config/forms/NavigationBarConfigForm.js]: just checked @ichalagashvili and looks like it is a better solution!
thanks!

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1065#pullrequestreview-654425990" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
